examples/image_classification/ios/ImageClassifier/Helper/CameraFeedManager.swift

```diff
@@ -0,0 +1,406 @@
+    NotificationCenter.default.addObserver(self, selector: #selector(CameraFeedManager.sessionWasInterrupted(notification:)), name: NSNotification.Name.AVCaptureSessionWasInterrupted, object: session)
+    NotificationCenter.default.addObserver(self, selector: #selector(CameraFeedManager.sessionInterruptionEnded), name: NSNotification.Name.AVCaptureSessionInterruptionEnded, object: session)
+  }
+
+  private func removeObservers() {
+    NotificationCenter.default.removeObserver(self, name: NSNotification.Name.AVCaptureSessionRuntimeError, object: session)
+    NotificationCenter.default.removeObserver(self, name: NSNotification.Name.AVCaptureSessionWasInterrupted, object: session)
+    NotificationCenter.default.removeObserver(self, name: NSNotification.Name.AVCaptureSessionInterruptionEnded, object: session)
+  }
+
+  // MARK: Notification Observers
+  @objc func sessionWasInterrupted(notification: Notification) {
+
+    if let userInfoValue = notification.userInfo?[AVCaptureSessionInterruptionReasonKey] as AnyObject?,
+       let reasonIntegerValue = userInfoValue.integerValue,
+       let reason = AVCaptureSession.InterruptionReason(rawValue: reasonIntegerValue) {
+      print("Capture session was interrupted with reason \(reason)")
+
+      var canResumeManually = false
+      if reason == .videoDeviceInUseByAnotherClient {
+        canResumeManually = true
+      } else if reason == .videoDeviceNotAvailableWithMultipleForegroundApps {
+        canResumeManually = false
+      }
+
+      self.delegate?.sessionWasInterrupted(canResumeManually: canResumeManually)
+
+    }
+  }
+
+  @objc func sessionInterruptionEnded(notification: Notification) {
+
+    self.delegate?.sessionInterruptionEnded()
+  }
+
+  @objc func sessionRuntimeErrorOccured(notification: Notification) {
+    guard let error = notification.userInfo?[AVCaptureSessionErrorKey] as? AVError else {
+      return
+    }
+
+    print("Capture session runtime error: \(error)")
+
+    if error.code == .mediaServicesWereReset {
+      sessionQueue.async {
+        if self.isSessionRunning {
+          self.startSession()
+        } else {
+          DispatchQueue.main.async {
+            self.delegate?.sessionRunTimeErrorOccured()
+          }
+        }
+      }
+    } else {
+      self.delegate?.sessionRunTimeErrorOccured()
+
+    }
+  }
+}
+
+
+/**
+ AVCaptureVideoDataOutputSampleBufferDelegate
+ */
+extension CameraFeedManager: AVCaptureVideoDataOutputSampleBufferDelegate {
+
+  /** This method delegates the CVPixelBuffer of the frame seen by the camera currently.
+   */
+  func captureOutput(_ output: AVCaptureOutput, didOutput sampleBuffer: CMSampleBuffer, from connection: AVCaptureConnection) {
+
+    // Converts the CMSampleBuffer to a CVPixelBuffer.
+    var pixelBuffer = CMSampleBufferGetImageBuffer(sampleBuffer)
```

Comment:
@khanhlvg @st-tuanmai 
Conversion is not necessary. Directly pass `sampleBuffer` according to the suggestion here.
https://github.com/googlesamples/mediapipe/pull/169/files#r1256173368